.github/workflows/ci.yml

```diff
@@ -173,27 +173,59 @@ jobs:
             macos-13,
             macos-latest,
           ]
-        musl: ["", "musllinux"]
-        exclude:
-          - os: windows-2019
-            musl: "musllinux"
-          - os: macos-13
-            musl: "musllinux"
-          - os: macos-latest
-            musl: "musllinux"
-
+        qemu: ['']
+        musl: [""]
+        include:
+          - os: ubuntu-latest
+            qemu: ppc64le
+            musl: ""
+          - os: ubuntu-latest
+            qemu: ppc64le
+            musl: musllinux
+          - os: ubuntu-latest
+            qemu: s390x
+            musl: ""
+          - os: ubuntu-latest
+            qemu: s390x
+            musl: musllinux
+          - os: ubuntu-latest
+            qemu: armv7l
+            musl: ""
```

Comment:
not sure if these images work but we can always remove it and try again in a followup if needed